e2eshark/onnx/models/opt-125M-awq/model.py

```diff
@@ -16,34 +16,92 @@
 # to form runmodel.py in the rundirectory which is then taken
 # through flow
 
-# set some variable parameters (dynamic dims)
-batch_size = 3 
-sequence_length = 7 
-past_sequence_length = 0 # setting this parameter to anything other than 0 results in an error
+# If you want to run the test with a static model.onnx, set run_as_static = True
+run_as_static = False
+dim_params = (
+    "batch_size",
+    "sequence_length",
+    "past_sequence_length",
+    "past_sequence_length + 1",
+)
+dim_values = (12, 7, 0, 1)
+pv_zip = zip(dim_params, dim_values)
+pv = dict(pv_zip)
+
+model = onnx.load("model.onnx")
+isDynamic = bool(model.graph.input[0].type.tensor_type.shape.dim[0].dim_param)
```

Comment:
match naming style https://peps.python.org/pep-0008/#function-and-variable-names
```suggestion
is_dynamic = bool(model.graph.input[0].type.tensor_type.shape.dim[0].dim_param)
```